.github/workflows/circuit-collator-release.yml

```diff
@@ -115,27 +103,51 @@ jobs:
 
       - name: 🎱 Gzip circuit parachain
         run: |
-          gzip -c ./target/release/parachain-collator \
-          > ./target/release/parachain-collator.gz
+          gzip -c ./target/release/circuit-collator \
+          > ./target/release/circuit-collator.gz
 
       - name: 🖐🏾 Fingerprint the gball
         run: |
-          sha256sum -b ./target/release/parachain-collator.gz \
+          sha256sum -b ./target/release/circuit-collator.gz \
           | grep -oE '^[a-f0-9]+' \
           | tr -d '\n' \
-          > ./target/release/parachain-collator.gz.sha256sum
+          > ./target/release/circuit-collator.gz.sha256sum
 
       - name: 🏷️ Get the version tag
         run: echo "PUSHED_TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
 
+      - name: 🖨️ Get the release notes
+        run: |
+          node -e "
+            const lines = require('fs').readFileSync('./CHANGELOG.md', 'utf8').split(/(\n)/g).filter(Boolean);
```

Comment:
Are we generating a CHANGELOG.md?

Reply:
no, just parsing it here to add it to the github release. requires to add release notes to the root changelog file before actually pushing the tag.